test/scripts/status.test.js

```diff
@@ -203,34 +206,49 @@ contract('status command', function([_, owner]) {
         await init({ name: appName, version, packageFileName });
         await addImplementation({ contractsData, packageFileName });
         await push({ packageFileName, network, txParams });
-        await status({ network, packageFileName, networkFileName, logger });
+        await status({ network, packageFileName, networkFileName });
 
-        logger.text.should.match(/no proxies/i);
+        this.logs.text.should.match(/no proxies/i);
       });
 
       it('should log created proxies', async function () {
         await init({ name: appName, version, packageFileName });
         await addImplementation({ contractsData, packageFileName });
         await push({ packageFileName, network, txParams });
         await createProxy({ contractAlias, network, txParams, packageFileName, networkFileName });
-        await status({ network, packageFileName, networkFileName, logger });
+        await status({ network, packageFileName, networkFileName });
         
-        logger.text.should.match(/Impl at 0x[0-9a-fA-F]{40} version 0.1.0/i);
+        this.logs.text.should.match(/Impl at 0x[0-9a-fA-F]{40} version 0.1.0/i);
       });
     });
   };
 
+  // Regression for https://github.com/zeppelinos/zos-cli/issues/191
```

Comment:
do we need this comment?

Reply:
I'd say so. It's an odd test to have (do not log something), so it's better if we add a clarification on why it was added.